operations/jsonnet/microservices/rollout-operator.libsonnet

```diff
@@ -0,0 +1,73 @@
+{
+  local k = import 'ksonnet-util/kausal.libsonnet',
+
+  local container = k.core.v1.container,
+  local deployment = k.apps.v1.deployment,
+  local policyRule = k.rbac.v1.policyRule,
+  local role = k.rbac.v1.role,
+  local roleBinding = k.rbac.v1.roleBinding,
+  local serviceAccount = k.core.v1.serviceAccount,
+  local podDisruptionBudget = k.policy.v1.podDisruptionBudget,
+
+  _config+:: {
+    rollout_operator_enabled: $._config.block_builder_concurrent_rollout_enabled,
+  },
+
+  rollout_operator_args:: {
+    'kubernetes.namespace': $._config.namespace,
+  },
+
+  rollout_operator_container::
```

Comment:
There is already a rollout-operator deployment in 

https://github.com/grafana/tempo/blob/545ab11c71f11f363b2f4a5b86fd4766ba9a52c7/operations/jsonnet/microservices/multi-zone.libsonnet#L146

Should we merge both?

Reply:
Yes! I had forgotten, thanks!